src/Control/Effect/Interpret.hs

```diff
@@ -1,20 +1,74 @@
-{-# LANGUAGE GeneralizedNewtypeDeriving, MultiParamTypeClasses, RankNTypes, TypeOperators, UndecidableInstances #-}
+{-# language FlexibleContexts #-}
+{-# language FlexibleInstances #-}
+{-# language FunctionalDependencies #-}
+{-# language GeneralizedNewtypeDeriving #-}
+{-# language KindSignatures #-}
+{-# language PolyKinds #-}
```

Comment:
1. I don’t think `PolyKinds` is being used?
2. `PolyKinds` implies `KindSignatures`.

So one or the other of these can go.

Reply:
`PolyKinds` was needed at one point, now seems fine without it. Removed.